modules/monitoring-querying-metrics-for-all-projects-with-mon-dashboard.adoc

```diff
@@ -4,8 +4,11 @@
 // * virt/support/virt-prometheus-queries.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="querying-metrics-for-all-projects-as-an-administrator_{context}"]
-= Querying metrics for all projects as a cluster administrator
+[id="querying-metrics-for-all-projects-with-mon-dashboard_{context}"]
+// The following section will be included in the administrator section, hence there is no need to include "administrator" in the title
+= Querying metrics for all projects with the {product-title} web console
```

Comment:
Suggest moving the comment down so it doesn't change the standard metadata layout
```suggestion
[id="querying-metrics-for-all-projects-with-mon-dashboard_{context}"]
= Querying metrics for all projects with the {product-title} web console
// The following section will be included in the administrator section, hence there is no need to include "administrator" in the title
```